_episodes/06-blurring.md

```diff
@@ -115,7 +115,18 @@ an average of the the colour values of the pixels surrounding it is calculated,
 weighted by the values in the kernel.
 In a Gaussian blur, the pixels nearest the centre of the kernel are
 given more weight than those far away from the centre.
-This averaging is done on a channel-by-channel basis,
+The rate at which this weight diminishes is determined by a Gaussian function, hence the name
+Gaussian blur. 
+
+A Gaussian function maps random variables into a normal distribution or "Bell Curve".
+![Gaussian function](../fig/1920px-Normal_Distribution_PDF.svg.png)
+The shape of the function is described by a mean value μ, and a variance value σ². The mean determines the central point of the bell curve on the x axis, and the variance describes the spread of the curve. 
+
+In fact, when using Gaussian functions in Gaussian blurring, we use a 2D Guassian function to account for X and Y dimensions, but the same rules apply. The mean μ is always 0, and represents the middle of the 2D kernel. Increasing values of σ² in either dimension increases the amount of blurring in that dimension. 
+
+![2D Gaussian function](../fig/1920px-Gaussian_2d_surface.png)
```

Comment:
Unfortunately this image is licensed CC-BY-SA and as such is not compatible with the CC-BY license of the lesson. It should be replaced with another image (CC-BY or CC0 licensed) or left out altogether.

Reply:
I'm assuming public access is ok?

Reply:
Anything in the public domain (i.e. CC0) is great.

Reply:
Although not required for a CC0 image, I would like to include attribution for images not created by the lesson authors (see my suggested change for the other image above) - please add a note about the source of the figure.

Reply:
Cool. I push the update to this pull request, but got an error from github relating to an automated build. However, it runs as expected on my dev machine.

Reply:
Yes. All the builds seem to be failing for PRs to this lesson today. It definitely is not an issue with your branch speicfically.

Reply:
I think the failed builds may be related to recent updates of the workflow template in #234, so nothing to worry about. Apologies for the confusion.